internal/generator/vector/output/otlp/transform.go

```diff
@@ -0,0 +1,254 @@
+package otlp
+
+import (
+	. "github.com/openshift/cluster-logging-operator/internal/generator/framework"
+	"github.com/openshift/cluster-logging-operator/internal/generator/vector/elements"
+	"github.com/openshift/cluster-logging-operator/internal/generator/vector/helpers"
+	"strings"
+)
+
+// VRL for OTLP transforms by route
+const (
+	CreateResourceAttributes = `
```

Comment:
Is here any advantage to convert this to VRL files and use embed?

Reply:
I would strongly suggest "not now".    Lets merge a working version before refactoring.